components/address/README.md

```diff
@@ -0,0 +1,7 @@
+## Address
+This pattern has now been deprecated. 
+
+The `uppercase` and reduced `font-size` styling is no longer a requirement for displaying addresses. 
+
+It should be replaced within services using `<p>` tags and any relevant utility classes.
+
```

Comment:
Maybe supplement with:

```suggestion
The styling will be removed in a future release (v2.0) of the pattern library. The .scss for address has been left in situ to avoid breaking changes at this time.
```